src/lib/without_body.rs

```diff
@@ -33,6 +34,7 @@ pub fn forward_request_without_body(
         .finish()
         .unwrap()
         .send()
+        .timeout(Duration::from_secs(300))
```

Comment:
Feel free to change it. We can set some value but for some people still it can be 0.X or 1 sec not enough. Unless we make it configurable. Sure it's extreme but still it's not unlimited value ;)